ambry-router/src/test/java/com.github.ambry.router/PutManagerTest.java

```diff
@@ -878,6 +879,17 @@ public long getSize() {
     return returnedFuture;
   }
 
+  @Override
+  public void setDigestAlgorithm(String digestAlgorithm)
```

Comment:
For these two, it doesn't matter actually (since they don't call the functions). Either approach is fine. I'll make it more consistent by throwing in both (since I don't expect them to be called). What do you think?